.github/workflows/build-and-push-docker.yml

```diff
@@ -0,0 +1,45 @@
+name: Docker build and push
+
+on:
+  push:
+    branches: ['v*.*.*']
+
+jobs:
+  docker:
+    name: docker build and push
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout 🙂
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Create image and tag names 😊
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: terrestris/geoserver
+          tags: |
+            type=ref,event=branch
+          flavor: |
+            latest=auto
+      - name: Generate docker tag 😝
+        run: echo "docker_tag=$(echo ${{ steps.meta.outputs.tags }} | sed 's|^terrestris/geoserver:v|terrestris/geoserver:|')" >> $GITHUB_ENV
+        id: generate-docker-tag
```

Comment:
You can also use the above metadata action for this https://github.com/marketplace/actions/docker-metadata-action#typesemver but of course this will work as well.

Reply:
really nice hint. i will try to use this now. thx

Reply:
main problem currently is that we are using branches but now tags. but i will try to adjust the cronjob script to use tags instead of branches in future